ReactWindows/ReactNative/UIManager/BorderedViewParentManager.cs

```diff
@@ -12,17 +12,53 @@ namespace ReactNative.UIManager
     public abstract class BorderedViewParentManager<TFrameworkElement> : ViewParentManager<Border>
         where TFrameworkElement : FrameworkElement
     {
+        private enum Radius
+        {
+            All,
```

Comment:
All = 0 (typical way to make it more explicit that people shouldn't change the enum ordering)